AppCenterReactNativeShared/Products/AppCenterReactNativeShared.podspec

```diff
@@ -8,7 +8,7 @@ Pod::Spec.new do |s|
 
   s.author           = { 'Microsoft' => 'appcentersdk@microsoft.com' }
 
-  s.source = { :http => "https://github.com/Microsoft/AppCenter-SDK-React-Native/releases/download/#{s.version}/AppCenter-SDK-ReactNative-iOS-Pod-#{s.version}.zip" }
```

Comment:
Please revert this change, this was only for publishing to private spec repo.

Reply:
Requested changes done.